platforms/access-bridge/src/services/plans-service.ts

```diff
@@ -0,0 +1,93 @@
+import { AccessControlPlan, AccessControlPlansParams, PlansResponse } from '@jwp/ott-common/types/plans.js';
+
+import { SIMS_CLIENT } from '../app-config.js';
+import { BadRequestError, ForbiddenError, NotFoundError, UnauthorizedError, isJWError } from '../errors.js';
+import { get } from '../http.js';
+
+/**
+ * Service class responsible for interacting with the Plans API that handles access control plans.
+ */
+export class PlansService {
+  private simsClient: string;
+
+  constructor() {
+    this.simsClient = SIMS_CLIENT;
+  }
+
+  /**
+   * Retrieves access control plans based on the specified endpoint type.
+   *
+   * Depending on the `endpointType` parameter, this function either:
```

Comment:
I would probably split these 2 endpoints into 2 separate methods. Does the 'plans' endpoint need to be in the middleware or could it be called from the FE directly?

Reply:
On the other hand, I did split them and made the services to be straight forward without any logic in them. All the logic goes in the controllers now. Thanks!